libagent/ssh/__init__.py

```diff
@@ -304,7 +314,29 @@ def main(device_type):
         with context:
             return run_server(conn=conn, command=command, sock_path=sock_path,
                               debug=args.debug, timeout=args.timeout)
+    elif args.sign_blob is not None:
+        if args.ecdsa_curve_name != 'ed25519':
+            print('Error: --sign-blob requires ed25519 curve')
+            return -1
+        blob = bytearray.fromhex(args.sign_blob)
+        identity = identities[0]
+        sig = conn.mc_blob_sign(blob=blob, identity=identities[0])
+        print('blob signature:')
+        print(sig.hex())
+    elif args.pemout is not None:
+        if args.ecdsa_curve_name != 'ed25519':
+            print('Error: --pemout requires ed25519 curve')
+            return -1
+        for pk in conn.public_keys():
+            with open(args.pemout, 'w') as p:
+                _,pubkey_sshenc_b64,_ = pk.split(' ')
+                pubkey= base64.b64decode(pubkey_sshenc_b64)[-32:]
+                pem_prefix = bytearray.fromhex('302a300506032b6570032100')
```

Comment:
would be nice to have some docu for this part, not immediately obvious to me what it means
i'm guessing that it's DER stuff that specifies that this is an ed25519 public key?
a code comment to that effect would help